app/src/main/java/com/codesoom/assignment/controllers/TasksController.java

```diff
@@ -0,0 +1,60 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.TaskManager;
+import com.codesoom.assignment.models.Task;
+import java.util.Collection;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * 할일에 대한 처리를 하는 컨트롤러.
```

Comment:
이름이 컨트롤러인데 컨트롤러라고 소개하면 좀 어색하겠죠.
```suggestion
 * 할 일에 대한 처리를 담당합니다.
```

Reply:
반영하였습니다!